Makefile

```diff
@@ -1,20 +1,18 @@
 .PHONY: test install
 
-all:	test
+all: install test
 
 test:
-	cd core && go get golang.org/x/tools/cmd/stringer && go generate && go test
-	cd crew && go test 
-	cd tools && go test
-	cd interpreters/goja && go test
-	cd cmd/patmatch && go test
-	cd cmd/mservice/storage/bolt && go test
-	cd cmd/mservice && go test
-	# cd cmd/mexpect && go test # Requires mservice executable
-	cd cmd/spectool && go test
+	@make install
```

Comment:
`install` could be a prerequisite of the `test` target instead of calling `make` inside the recipe.